btree/btree.c

```diff
@@ -3466,6 +3489,7 @@ static int64_t btree_put_makespace_phase(struct m0_btree_op *bop)
 
 		node_move(lev->l_alloc, lev->l_node, D_RIGHT,
 		          NR_MAX, bop->bo_tx);
+		lev->i_alloc_in_use = false;
```

Comment:
Uncomment `node_capture` in btree_tx_node_capture().

Reply:
done.